blazor/dropdown-list/selection.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: Selection in Blazor DropDown List Component | Syncfusion
+description: Checkout and learn here all about Selection feature in Syncfusion Blazor DropDownList component and more.
+platform: Blazor
+control: DropDown List
+documentation: ug
+---
+
+# Selection
+
+## Get Selected Value
+
+You can get the selected value of the DropDownList component from the `args.Value` property in the [ValueChange](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.ChangeEventArgs-2.html#Syncfusion_Blazor_DropDowns_ChangeEventArgs_2_Value)  event argument. 

```

Comment:
You can get the selected value of the DropDownList component in [ValueChange]() event using [ChangeEventArgs.Value]() property.